### PR TITLE
Add Gene Query to ResultsViewPage

### PR DESCRIPTION
**Pull request by Luke-Sikina**
Changes proposed in this pull request:
- Created callback that updates url with query
- Added GeneSelectionBox
- Did very minimal styling

# Checks
- [x] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).

# Any screenshots or GIFs?
Screenshot
# Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). If you are part of the
cBioPortal organization, notify the approprate team (remove inappropriate):

@cBioPortal/frontend

### CODE REVIEW

**Comment by zhx828** [comment]
While Luke at the topic,  @Luke-Sikina @alisman @kalletlak could we take a look this ticket? https://github.com/cBioPortal/cbioportal/issues/5378

File: src/pages/resultsView/styles.scss [hunk 1]
```diff
@@ -111,4 +111,8 @@
   .alert {
     margin-bottom:5px;
   }
+}
+
+.queryButton {
```
Comment (alisman) [inline src/pages/resultsView/styles.scss]: need to fill you in on the way styles and css modules work

**Comment by Luke-Sikina** [comment]
Block from merging pending feedback from product team.

**Comment by Luke-Sikina** [comment]
Redoing.

**Pull request closed without merging**